### ISSUE TITLE
Mute feature

### ISSUE THREAD

**Issue by SGoeR** [SoundControl/Issues#97]
Mute is not turning off after the commercials so it is muting the next songs until I manually unmute it.

**Comment by SoundControl** [comment]
Please explain your problem further. is it:
1) song > ad > song, volume was not brought back to normal
or
2) song > ad, maybe restart the browser, pause, whatever > song, volume is
at an ad level?

**Comment by SGoeR** [comment]
I have the option set to play the ads at 10%.  The music plays at normal
volume and then an ad comes on and play at 10%.  I didn't notice it for a
while but then I realized that I hadn't heard anything for a few minutes so
I went to the Pandora tab and the song is playing but there is a button on
the album art that says "muted".  If I click the "muted" button the sound
comes back on at normal volume.  When the next ad comes on the same things
happens again.  I have used SoundControl for probably a year now and this
just started happening in the last few days.

Right now I am just listening to the ads at normal volume. I know, the
horror :)  Thanks for your help.
